src/internet_identity/tests/tests.rs

```diff
@@ -64,11 +64,7 @@ mod upgrade_tests {
             &env,
             canister_id,
             II_WASM.clone(),
-            Some(InternetIdentityInit {
```

Comment:
I extracted the helper function, because I had to touch all these locations anyways because of the new field in the init argument.